src/index.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import App from './App';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import routes from './routes';
+
+const router = createBrowserRouter(routes);
```

Comment:
I think that `createBrowserRouter` statement could be moved to `routes.tsx` file so that `index.tsx` stays cleaner